test/src/Tests/Controller/SampleControllerXmlTest.php

```diff
@@ -61,11 +61,12 @@ public function testGetResponseFromMockedService()
 
     public function testProductIndexResponse()
     {
-        $this->loadFixturesFrom('product.yml');
+        $this->loadFixturesFromSingleFile('product.yml');
```

Comment:
I would suggest to load here files from default directory. We have only one file there, so the result will be same, but we would check if it works properly.